server/src/app/index.js

```diff
@@ -0,0 +1,49 @@
+import express from 'express';
+import index from './routes/index';
+import users from './routes/users';
+import mongoose, { mongo } from 'mongoose';
+import session from 'express-session';
+import passport from 'passport';
+import configPassport from './middlewares/passport';
+import dotenv from 'dotenv';
+
+dotenv.config({ path: '../.env' });
+
+configPassport(passport);
+
+const port = process.env.PORT || 5000;
+
+const app = express();
+
```

Comment:
We should also configure express to parse JSON by default as `app.use(express.json())` automatically. It will be handy.

Reply:
Sure....